requirements.txt

```diff
@@ -1,7 +1,5 @@
 gunicorn==19.9.0
 
-git+https://github.com/CodeForAfrica/HURUmap.git@master#egg=hurumap
```

Comment:
1. `gunicorn[gevent]` is already installed in HURUmap image,
 2. `git+https://github.com/OpenUpSA/wazimap.git@releases#egg=wazimap` shouldn't be here, `https://github.com/CodeForAfricaLabs/wazimap.git@openup/releases#egg=wazimap[gdal]` is installed in HURUmap image.
 3. We also need `https://github.com/CodeForAfricaLabs/wazimap.git@master#egg=wazimap` due to the URLs fix we did (otherwise country slugs with a `-` won't work.
![screenshot from 2019-02-19 16-23-32](https://user-images.githubusercontent.com/1779590/53018180-b8993b00-3462-11e9-9544-3237d9e8ebc3.png)

Reply:
> * `gunicorn[gevent]` is already installed in HURUmap image,
> * `git+https://github.com/OpenUpSA/wazimap.git@releases#egg=wazimap` shouldn't be here, `https://github.com/CodeForAfricaLabs/wazimap.git@openup/releases#egg=wazimap[gdal]` is installed in HURUmap image.
> * We also need `https://github.com/CodeForAfricaLabs/wazimap.git@master#egg=wazimap` due to the URLs fix we did (otherwise country slugs with a `-` won't work.
>   ![screenshot from 2019-02-19 16-23-32](https://user-images.githubusercontent.com/1779590/53018180-b8993b00-3462-11e9-9544-3237d9e8ebc3.png)

@kilemensi I think 3 doesn't work coz that page doesn't exist, the urls fix is also in Takwimu urls https://github.com/TakwimuAfrica/Takwimu/blob/be3fe30560b01202c7c988e33392aa40b9ff19ce/takwimu/urls.py#L16

Reply:
That's even better @phillipahereza ... so just 1 & 2

Reply:
@kilemensi 1 and 2 already fixed

Reply:
Spoke too soon @phillipahereza ....

 1. `http://takwimu.africa/profiles/south-africa/` works so `http://localhost:8000/profiles/south-africa/` should work, and
 2. Even if its not there, the error shouldn't be `GeographyDetailView`. This tells me it was wazimap and not wagtail. Compare to this:
![screenshot from 2019-02-19 16-59-42](https://user-images.githubusercontent.com/1779590/53020353-c7362100-3467-11e9-8a13-779f0fb5fe08.png)
Fictional `nyau-nyau` throws _wagtail_ error.

Reply:
@kilemensi found it https://github.com/CodeForAfrica/HURUmap/blob/d66113d48b7b74c0902183e1bf5458d5ba04bd7c/hurumap/dashboard/urls.py#L15
since the hurumap_urls appear before the dashboard urls, it errors in the wazimap geography on in there. Let me fix it in HURUmap